scripts/wrappers/status.py

```diff
@@ -34,9 +36,14 @@ def print_pretty(isReady, enabled_addons, disabled_addons):
     console_formatter = "{:>3} {:<20} # ({}) {}"
     if isReady:
         print("microk8s is running")
+        etcd_endpoints = get_etcd_info()
         if not is_ha_enabled():
             print("high-availability: no")
-        else:
+            if etcd_endpoints:
+                print("{:>2}{}".format("", "datastore endpoints:"))
+                for endpoint in etcd_endpoints:
```

Comment:
Here is what this is printing:
```
microk8s is running
high-availability: no
  datastore endpoints:
   127.0.0.1:12379
addons:
  enabled:
    helm                 # (core) Helm - the package manager for Kubernetes
    helm3                # (core) Helm 3 - the package manager for Kubernetes
  disabled:
.....
```

Have a look at the indentation of 127.0.0.1. I bet people are parsing this output and this will break them.

Reply:
This is intentional, as there can be multiple URLs like this:
```
--etcd-servers=https://10.139.219.7:2379,https://10.139.219.148:2379,https://10.139.219.100:2379
```

Reply:
We are missing a space before `127.0.0.1`.

Reply:
Done!